lib/crypto/butil.js

```diff
@@ -35,7 +35,7 @@ module.exports.xorBuffers = function xorBuffers(buffer1, buffer2) {
 module.exports.unbuffer = function unbuffer(object, inplace) {
   var keys = Object.keys(object)
   var copy = inplace ? object : {}
-  for (var i = 0; i < keys.length; i++) {
+  for (var i = 0; i < keys.length; i ++) {
```

Comment:
@divyabiyani - ISTM there are multiple PRs to be made out of this one. The first PR is to use the `no new buffer` eslint rule and update any place that's required. 

The 2nd is the more contentious issue of no-space in unary operators except `!` and `!!`, which should probably be discussed with the group before merging.